packages/react-ui-ag/src/Listings/MobileSRPListing.js

```diff
@@ -0,0 +1,82 @@
+import React, { PureComponent } from 'react'
+import PropTypes from 'prop-types'
+import themed from 'react-themed'
+import classnames from 'classnames'
+import { ListingComponents, Schema } from '@rentpath/react-ui-core'
+import Listing from './Listing'
+
+@themed([
+  'MobileSRPListing',
+  'BedsAndBaths',
+  'UnitLevelAvailabilityAndLastUpdated',
+  'LastUpdated',
+  'Phone',
+], { pure: true })
+export default class MobileSRPListing extends PureComponent {
```

Comment:
Can we call this something that doesn't have SRP in the name? If it's something that can be reused across mobile how about just `MobileListing`?

Reply:
Yeah no problem. There is already a generic `MobileListing`, but I'm confused because I don't think we actually use it anywhere